### ISSUE TITLE
ValueError: Protocol message Document has no "type" field.

### ISSUE THREAD

**Issue by Arthur-Cassarin-Grand** [googleapis/python-language#43]
Hello,

I've got an issue since I've passed my project to google-cloud-language 2.0.0.
I tried on the Cloud Shell Protocol offered by the [official documentation](https://cloud.google.com/natural-language/docs/analyzing-syntax?hl=en). This bug appears everytime.

#### Environment details

  - OS type and version: MacOS - Catalina
  - Python version: Python 3.7.0
  - pip version: pip 20.2.4
  - `google-cloud-language` version: 2.0.0

**pip freeze**

asgiref==3.2.10
cachetools==4.0.0
certifi==2019.11.28
chardet==3.0.4
colorama==0.4.4
cygrpc==1.0.4.post6
google-api-core==1.23.0
google-api-python-client==1.12.4
google-auth==1.22.1
google-auth-httplib2==0.0.4
google-cloud==0.34.0
google-cloud-language==2.0.0
googleapis-common-protos==1.6.0
grpcio==1.31.0
grpcio-tools==1.23.0
idna==2.8
libcst==0.3.13
oauth2client==4.1.3
packaging==20.4
pkginfo==1.6.0
proto-plus==1.11.0
protobuf==3.13.0

#### Steps to reproduce

  1. I use the sample code of the library here : samples/v1/language_syntax_text.py
  2. python3 samples/v1/language_syntax_text.py

#### Code example

```python
def gcl_analyze_syntax(sentence):
    client = language_v1.LanguageServiceClient()
    type_ = language_v1.Document.Type.PLAIN_TEXT
    language = "fr"
    document = {"content": sentence, "type": type_, "language": language}
    encoding_type = language_v1.EncodingType.UTF8
    tokens = client.analyze_syntax(request = {'document': document, 'encoding_type': encoding_type}).tokens
    return tokens
```
or

```python
def sample_analyze_syntax(text_content):
    """
    Analyzing Syntax in a String
    Args:
      text_content The text content to analyze
    """

    client = language_v1.LanguageServiceClient()

    # text_content = 'This is a short sentence.'

    # Available types: PLAIN_TEXT, HTML
    type_ = language_v1.Document.Type.PLAIN_TEXT

    # Optional. If not specified, the language is automatically detected.
    # For list of supported languages:
    # https://cloud.google.com/natural-language/docs/languages
    language = "en"
    document = {"content": text_content, "type": type_, "language": language}

    # Available values: NONE, UTF8, UTF16, UTF32
    encoding_type = language_v1.EncodingType.UTF8

    response = client.analyze_syntax(request = {'document': document, 'encoding_type': encoding_type})
```

#### Stack trace
```
Traceback (most recent call last):
  File "samples/v1/language_syntax_text.py", line 110, in <module>
    main()
  File "samples/v1/language_syntax_text.py", line 106, in main
    sample_analyze_syntax(args.text_content)
  File "samples/v1/language_syntax_text.py", line 54, in sample_analyze_syntax
    response = client.analyze_syntax(request = {'document': document, 'encoding_type': encoding_type})
  File "/usr/local/lib/python3.7/dist-packages/google/cloud/language_v1/services/language_service/client.py", line 533, in analyze_syntax
    request = language_service.AnalyzeSyntaxRequest(request)
  File "/usr/local/lib/python3.7/dist-packages/proto/message.py", line 461, in __init__
    pb_value = marshal.to_proto(pb_type, value)
  File "/usr/local/lib/python3.7/dist-packages/proto/marshal/marshal.py", line 205, in to_proto
    pb_value = rule.to_proto(value)
  File "/usr/local/lib/python3.7/dist-packages/proto/marshal/rules/message.py", line 32, in to_proto
    return self._descriptor(**value)
ValueError: Protocol message Document has no "type" field.
```

Thanks,
Arthur

**Comment by santma** [comment]
I am having the same problem. Let me know if you salved this somehow. Thanks!

**Comment by paukiun** [comment]
i have the same question. And when i downgrade my google.cloud.language to 1.3.0, python still went wrong saying "do not have the attribute document"

**Comment by Arthur-Cassarin-Grand** [comment]
Okay thank you @busunkim96.

For those who didn't have understand, here's the code fixed.

```python
def gcl_analyze_syntax(sentence):
    client = language_v1.LanguageServiceClient()
    type_ = language_v1.Document.Type.PLAIN_TEXT
    language = "fr"
    document = {"content": sentence, **_"type_": type__**, "language": language}
    encoding_type = language_v1.EncodingType.UTF8
    tokens = client.analyze_syntax(request = {'document': document, 'encoding_type': encoding_type}).tokens
    return tokens
```

**Comment by busunkim96** [comment]
The samples and RST files need to be updated with this change.

**Comment by telpirion** [comment]
Fixed in #49 .
